packages/core/src/anchor/ethereum/ethereum-anchor-service.ts

```diff
@@ -302,6 +302,15 @@ export default class EthereumAnchorService implements AnchorService {
    * Parse JSON that CAS returns.
    */
   private parseResponse(cidStream: CidAndStream, json: any): AnchorServiceResponse {
+    if (json.error) {
```

Comment:
This is the change to take advantage of https://github.com/ceramicnetwork/ceramic-anchor-service/pull/433

Reply:
See https://github.com/ceramicnetwork/ceramic-anchor-service/pull/433#issuecomment-854017254

Not good.